src/store/modules/globalErrorsModule.js

```diff
@@ -31,7 +31,13 @@ function countErrors(obj) {
 }
 
 const updateValidationRules = async (mainScreen, commit) => {
-  await mainScreen.loadValidationRules();
+  try {
+    await mainScreen.loadValidationRules();
+  } catch (error) {
+    if (this.$store.getters["globalErrorsModule/getMode"] === "preview") {
```

Comment:
Consider to use `getters` as argument same as `commit` instead of `this.$store.getters["globalErrorsModule/getMode"]`
e.g.
```
const updateValidationRules = async (mainScreen, commit, getters) => {
...
      if (getters.getMode() === "preview") {
...
    async validateNow({ commit, getters }, mainScreen) {
      await updateValidationRules(mainScreen, commit, getters);
    },
```